client/src/components/RightPanel/chatRoom/chatRoomFooter/ChatRoomFooter.tsx

```diff
@@ -1,30 +1,34 @@
 
 import { useSocket } from "../../../../Context/SocketContext/SocketContext"
-import { IChatRoomProps } from "../../../../utils/interfaces";
+// import { IChatRoomProps } from "../../../../utils/interfaces";  // Används inte
 import "./ChatRoomFooter.css"
 import { useState } from "react";
 
-function ChatRoomFooter({ roomName }: IChatRoomProps) {
+function ChatRoomFooter(/*{ roomName }: IChatRoomProps*/) { // roomName Används inte
 
-    const { currentRoom, sendMessage } = useSocket()
+    const { currentRoom, sendMessage, username } = useSocket()
 
     const [message, setMessage] = useState('');
-    const [roomMessages, setRoomMessages] = useState<string[]>([]);
+    // const [roomMessages, setRoomMessages] = useState<string[]>([]); // Används inte
+
+    const timestamp = new Date(Date.now()).getHours() + ":" + new Date(Date.now()).getMinutes()  // < --- Tid för meddelandet
```

Comment:
We can display double digits even when time as the value of 1-9 by:

String(new Date(Date.now()).getMinutes()).padStart(2, "0")